www/webapp/main.py

```diff
@@ -15,9 +15,10 @@
 
 @webapp.route('/')
 def index():
-    return render_template("index.html")
+    return render_template('index.html', translations=get_translations())
```

Comment:
With the get_translations function we can add the page name as a string, to get only the translations we need